rocAL/rocAL/include/reader.h

```diff
@@ -23,54 +23,85 @@ THE SOFTWARE.
 #pragma once
 #include <string>
 #include <map>
+#include <vector>
+#include<tuple>
 #include "meta_data_reader.h"
 
 enum class StorageType
 {
     FILE_SYSTEM = 0,
     TF_RECORD = 1,
-    UNCOMPRESSED_BINARY_DATA = 2,        // experimental: added for supporting cifar10 data set
+    UNCOMPRESSED_BINARY_DATA = 2, // experimental: added for supporting cifar10 data set
     CAFFE_LMDB_RECORD = 3,
     CAFFE2_LMDB_RECORD = 4,
-    COCO_FILE_SYSTEM = 5
+    COCO_FILE_SYSTEM = 5,
+    VIDEO_FILE_SYSTEM = 6,
+    SEQUENCE_FILE_SYSTEM = 7
 };
 
 struct ReaderConfig
```

Comment:
better to use separate ReaderConfig for video and images.

Reply:
@fiona-gladwin : are we going to do this change in this PR? Keep video reader and image reader separate so image reader doesn't need to take video specific parameters